taxonomies/categories.txt

```diff
@@ -27286,29 +27286,11 @@ nl:Pralines, chocolades snoepjes
 pt:Bombons
 ru:Конфеты
 wikidata:en:Q3272827
+agribalyse_food_code:en:31091
+ciqual_food_code:en:31091
+ciqual_food_name:en:Chocolate confectionery, filled with wafer
```

Comment:
this CIQUAL category is for chocolate confectionery with a wafer filling, it's not any kind of candy, it does not work here.

Reply:
Maybe more this one: https://www.agribalyse.fr/app/aliments/31063#Bouchée_chocolat_fourrage_fruits_à_coques_et/ou_praliné

Reply:
in fact my mistake, Github is not showing the lines above (only 3), I didn't realize it was for chocolate bonbons.

so it's a good category. Make it a proxy maybe.

```suggestion
ciqual_proxy_food_name:en:Chocolate confectionery, filled with wafer
```